src/components/Layout/config.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+import { Outlet, NavLink, useMatch } from "react-router-dom";
+import clsx from "clsx";
+
+import { SearchLayout } from "./search";
+
+export function ConfigLayout({
+  showSearchInput,
+  basePath,
+  backPath,
+  navLinks,
+  ...props
+}) {
+  const [title, setTitle] = useState(props.title || "");
+  const [titleExtras, setTitleExtras] = useState();
+  const [tabRight, setTabRight] = useState();
+  const mt = useMatch({ path: basePath, end: false });
+
+  const nav = (
+    <nav
+      className="flex justify-between"
+      style={{
+        borderBottom: "solid 1px lightgray",
+        marginBottom: "1rem"
+      }}
+    >
+      <span className="flex self-center">
+        {navLinks.map((nav) => (
```

Comment:
nit: we can extract a new NavList component here.

Reply:
ack. will pickup in a separate MR.

Reply:
https://github.com/orgs/flanksource/projects/3#card-82549947 For tracking.